sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -74,7 +74,7 @@ def test_plot_quadrangle(aia171_test_map, plotter):
                            obstime=aia171_test_map.date)
     plotter.plot_quadrangle(bottom_left=bottom_left, width=20*u.deg,
                             height=60*u.deg, color='blue')
-    assert plotter.plotter.mesh.n_cells == 4001
+    assert plotter.plotter.mesh.n_cells == 4000
```

Comment:
What caused the number change?

Reply:
Ah, in `plot_quadrangle` I was adding the meshes directly after converting it to a numpy array, although this is fine because it is internally converted to a mesh, we needed to add a field array. Like plot_map, the numpy array is converted to a `StructuredGrid` and then a field array can be added, this caused a change in the number of cells.